cmd/metricswatcher/prometheushealthcheck/healthprovider.go

```diff
@@ -30,7 +30,7 @@ func NewHealthProvider(server string) (*HealthProvider, error) {
 		return nil, fmt.Errorf("error while starting healthcheck: %w", err)
 	}
 
-	url.Path = path.Join(url.Path, "/-/healthy")
+	url.Path = filepath.Join(url.Path, "/-/healthy")
```

Comment:
Actually, this usage was correct, since we're dealing with URLs. The documentation in the `path/filepath` package says:
> To process paths such as URLs that always use forward slashes regardless of the operating system, see the path package.